org.eclipse.scanning.test/src/org/eclipse/scanning/test/malcolm/AbstractMalcolmTest.java

```diff
@@ -102,6 +101,12 @@ protected void createParameters(Map<String, Object> config, long configureSleep,
 
 	}
 
+	protected IMalcolmDevice createMalcolmDevice(String name) throws MalcolmDeviceException {
+		IMalcolmDevice<?> device = new MalcolmDevice(name, connectorService, service, null); // Might throw exception
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "device" which hides the field declared at line 51. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AHiddenFieldCheck)